src/pages/Clusters.js

```diff
@@ -132,27 +132,27 @@ const Clusters = () => {
     size: 'size'
   }[diskSort.field]], [diskSort.direction], disks)
 
-  const totalCost = _.sum(_.map(clusterCost, clusters))
+  const totalCost = _.sum(_.map(clusterCost, runtimes))
```

Comment:
Presumably clusterCost is going to be changed with `cluster-utils`?

Reply:
Yes, this PR is scoped to just local names within one file for easier reviewing.